servant-quickcheck.cabal

```diff
@@ -93,7 +93,7 @@ test-suite spec
   other-modules:       Servant.QuickCheck.InternalSpec
   build-tool-depends:  hspec-discover:hspec-discover
   build-depends:       base
-                     , base-compat
+                     , base-compat-batteries
```

Comment:
I was getting a failure to load the test suite in GHCi  due to an ambiguous package. This resolved it.